CMakeLists.txt

```
@@ -211,3 +211,11 @@ install (TARGETS OpenCL
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+
+if (UNIX)
+  configure_file(OpenCL.pc.in OpenCL.pc @ONLY)
+  set(pkg_config_location ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+  install(
+    FILES ${CMAKE_CURRENT_BINARY_DIR}/OpenCL.pc
+    DESTINATION ${pkg_config_location})
+endif()
```

```suggestion
configure_file(OpenCL.pc.in OpenCL.pc @ONLY)
set(pkg_config_location ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
install(
  FILES ${CMAKE_CURRENT_BINARY_DIR}/OpenCL.pc
  DESTINATION ${pkg_config_location})
```
This removes the Unix check to enable `pkg-config` support for Windows.

---

Many projects are using `if (NOT MSVC)` as a guard for .pc file creation.

---

Thanks for the suggestion! Is there a reason not to install it on windows (see above discussion)? We could also make it optional and off by default on Windows, we would need to document that.